### PR TITLE
subsys/mgmt/hawkbit: Clean up flash map usage

### PR DESCRIPTION
**Pull request by de-nordic**
The commit cleans up how flash map labels and definitions are
used.

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

### CODE REVIEW

**Review by Navin-Sankar — DISMISSED** [review]
LGTM

File: subsys/mgmt/hawkbit/hawkbit.c [hunk 1]
```diff
@@ -572,13 +575,13 @@ int hawkbit_init(void)
...
 
-	fs.flash_device = DEVICE_DT_GET(FLASH_NODE);
+	fs.flash_device = STORAGE_DEV;
 	if (!device_is_ready(fs.flash_device)) {
 		LOG_ERR("Flash device not ready");
 		return -ENODEV;
 	}
 
-	fs.offset = FLASH_AREA_OFFSET(storage);
+	fs.offset = FLASH_AREA_OFFSET(STORAGE_LABEL);
```
Comment (ycsin) [inline subsys/mgmt/hawkbit/hawkbit.c:584]: Should be using the macro instead

**Review by ycsin — CHANGES_REQUESTED** [review]
Thanks for the patch, just 1 comment

File: subsys/mgmt/hawkbit/hawkbit.c (hunk 1 above)
Comment (de-nordic) [reply subsys/mgmt/hawkbit/hawkbit.c]: I have intentionally created the define to use here and somehow failed to do so. Fixed now.

**Review by ycsin — APPROVED** [review]

**Review by Navin-Sankar — APPROVED** [review]

**Pull request merged**